RazorHtmlEmails.RazorClassLib/RazorHtmlEmails.RazorClassLib.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
As I told before for me it should be in the same project as the api. We don't divide thing here (yet?) to infrastructure and application. Another DLL is not the advantage. It might be useful if we will have for example NotificationService which don't care how this the message gonna be send (eg. email/sms/notification). 

If DbContext is "Infrastructure" in our project why YesterdayApi.Core.Email is core.